### DISCUSSION TITLE
NDI Support

### DISCUSSION THREAD

**Discussion by zionashirwada** [ChurchApps/ChurchAppsSupport#317] [FreeShow] [answered]
What about giving an NDI output as a multiple screen ?

**Comment by vassbo** [comment]
I am trying to get NDI to work as mentioned here 😊: https://github.com/vassbo/freeshow/issues/165

**Comment by vassbo** [comment, marked as answer]
There is now NDI support for Windows and MacOS in version 0.9.5!

**Comment by TPaulManohar** [comment]
Can we optimise the NDI for Smooth transmission over Network, Because it taking lot of computer resources.  Some sort of settings like enable/disable screen recordings granular controls.

**Reply by vassbo** [reply]
You can reduce the framerate to make it use less computer/network resources!

**Reply by TPaulManohar** [reply]
We use 24fps
